### PR TITLE
Add notebook for Kaggle importers

### PR DESCRIPTION
**Pull request by wonjuleee**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [x] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: docs/source/docs/data-formats/formats/kaggle.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Kaggle
+
+## Format specification
+
+[Kaggle](https://www.kaggle.com/) provides more than 2800 computer vision datasets for the public good and fair competition.
+All datasets are available for downloading at [here](https://www.kaggle.com/datasets?tags=13207-Computer+Vision).
+However, since Kaggle doesn't enforce community to follow specific rule for dataset uploads, it is more natural to explore a dataset directoy structure by manual.
+So it eventually requires to take some time for importing those datasets into their machine learning codes.
+Therefore, Datumaro is providing ability to import them through Datumaro Python APIs.
```
Comment (vinnamkim) [inline docs/source/docs/data-formats/formats/kaggle.md]:
```suggestion
Therefore, Datumaro is providing an ability to import them through Datumaro Python APIs.
```

File: docs/source/docs/data-formats/formats/kaggle.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Supported type of annotations:
+- `Label` (classification)
+- `Bbox` (object detection)
+- `Mask` (segmentation)
+
+## Import Kaggle Image CSV dataset
+
+Indeed, Kaggle doesn't have any specific directory structure, and Datumaro hence requires more arguments for importing.
```
Comment (vinnamkim) [inline docs/source/docs/data-formats/formats/kaggle.md]:
```suggestion
Indeed, Kaggle doesn't have any specific directory structure, and Datumaro hence requires more user-assisted arguments for importing.
```

File: docs/source/docs/data-formats/formats/kaggle.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+A Datumaro dataset with a Kaggle dataset can be created in the following way in Python API:
+
+```python
+import datumaro as dm
+
+dataset = dm.Dataset.import_from('<path_to_image_directory>', format='kaggle_image_csv', ann_file='<path_to_csv_file>', columns={"media": "column_name_of_media", "label": "column_name_of_label"})
+```
+
```
Comment (vinnamkim) [inline docs/source/docs/data-formats/formats/kaggle.md:46]:
```suggestion
At this time, it's essential to specify the column names for media and label such as `dm.Dataset.import_from(..., columns={"media": "column_name_of_media", "label": "column_name_of_label"})
```

**Comment by wonjuleee** [comment]
<p>fixed at 2e68f4a8a4ae6c77b1321ab426b2cb3af55528ed</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/1254/discussion/#comment-1920630729'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by wonjuleee** [comment]
<p>fixed at 2e68f4a8a4ae6c77b1321ab426b2cb3af55528ed</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/1254/discussion/#comment-1920630745'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: docs/source/docs/data-formats/formats/kaggle.md (hunk 1 above)
Comment (wonjuleee) [reply docs/source/docs/data-formats/formats/kaggle.md]: fixed at 2e68f4a8a4ae6c77b1321ab426b2cb3af55528ed

File: docs/source/docs/data-formats/formats/kaggle.md (hunk 2 above)
Comment (wonjuleee) [reply docs/source/docs/data-formats/formats/kaggle.md]: fixed at 2e68f4a8a4ae6c77b1321ab426b2cb3af55528ed

File: docs/source/docs/data-formats/formats/kaggle.md (hunk 3 above)
Comment (wonjuleee) [reply docs/source/docs/data-formats/formats/kaggle.md:46]: fixed at 2e68f4a8a4ae6c77b1321ab426b2cb3af55528ed

**Comment by wonjuleee** [comment]
<p>It sounds great. Let me create a new requirement for this.</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/1254/discussion/#comment-1920630757'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Pull request merged**